code/modules/mob/living/carbon/human/life/handle_grabbed.dm

```diff
@@ -1,9 +1,7 @@
-
-
 /mob/living/carbon/proc/handle_grabbed()
```

Comment:
a file committed with ONLY whitespace changes.

Reply:
I'd say the whitespace at the top is the bigger issue than fixing it.

Reply:
In general we don't want to make the habit of just adding/removing whitespace formatting because then we go full OCD and back-and-forth on undefined standards. Like `if()` and `if ()`.
No need to change this back now, it'd be a bit too much, but try not to do it in the future.

Reply:
there is a difference between `if ()` and `if()` and random whitespace at the top